scripts/build-docs.js

```diff
@@ -1,154 +0,0 @@
-/* eslint-env node */
-'use strict';
-
-const path = require('path');
-const fs = require('fs-extra');
-
-const DEST_DIR = 'dist';
-const GITHUB_FORMAT = 'github';
-const KRAMDOWN_FORMAT = 'kramdown';
-const PC_DEV_CENTER_FORMAT = 'purecloudDevCenter';
-const SUPPORTED_DOC_OUTPUT_FORMATS = [GITHUB_FORMAT, KRAMDOWN_FORMAT, PC_DEV_CENTER_FORMAT];
-let docMdOutputFormat = process.env.DOC_MD_OUTPUT_FORMAT || PC_DEV_CENTER_FORMAT;
-if (SUPPORTED_DOC_OUTPUT_FORMATS.indexOf(docMdOutputFormat) < 0) {
-    console.error(`Unknown MD Output Format Specified: '${docMdOutputFormat}'`);
-    process.exit(1);
-}
-
-let linkExtension = '.md';
-if (docMdOutputFormat === PC_DEV_CENTER_FORMAT || docMdOutputFormat === KRAMDOWN_FORMAT) {
-    linkExtension = '.html';
-}
-
-let flagDocsAsPreview = true;
-if (process.env.FLAG_DOCS_AS_PREVIEW) {
-    ['false', 'f', '0'].forEach(currFalseValue => {
-        if (process.env.FLAG_DOCS_AS_PREVIEW === currFalseValue) {
-            flagDocsAsPreview = false;
-        }
-    });
-}
-
-// If invoked via "npm run docs" or "node scripts/build-docs.js" run immediately
-const calledFromCli = !module.parent;
-if (calledFromCli) {
-    buildDocs().catch(error => {
-        console.error('Documentation generation failed', error);
-        process.exit(1);
-    });
-}
-
-function buildDocHeader(title) {
-    let result = `---\ntitle: ${title}`;
-    if (flagDocsAsPreview) {
-        result += '\nispreview: true';
-    }
-    result += '\n---\n\n';
-
-    return result;
-}
-
-function transformRelativeLinks(buffer, ext) {
-    const RELATIVE_LINK_REPLACE_REGEXP = /\(\.\/([^)]+)\.[^)]+\)/gm;
-    return buffer.replace(RELATIVE_LINK_REPLACE_REGEXP, (match, name) => {
-        return `(./${name}${ext})`;
-    });
-}
-
-/**
- * Returns a copy of the github-flavored markdown srcBuffer transformed into the target format.
- *
- * @param {string} srcBuffer The base github-flavored markdown format
- * @param {string} targetFormat The desired markdwon format of the response
- */
-function transformGithubMarkdown(srcBuffer, targetFormat) {
```

Comment:
The code blocks look good when running the dev center locally.  Guess that means it now supports gh md code block syntax?

Reply:
Yep, that's the behavior I was seeing so I dropped this.